rdf-star-cg-spec.html

```diff
@@ -106,7 +106,7 @@ <h2>Background and Motivation</h2>
     <section class="informative">
       <h2>Overview</h2>
 
-<p>The RDF data model lets you state facts in three-part subject-predicate-object statements known as triples. For example, with a single RDF triple you can say that employee38 has a hireDate of "2020-04-03". A triple's predicate is a property specified with an IRI (an Internationalized version of a URI) to identify the namespace of the property name. A triple's subject and object can each be any entity that can be referenced using an IRI, and the object can also be a literal value such as "2020-04-03" or another data type such as a numeric or Boolean value. </p>
+<p>The RDF data model lets you state facts in three-part subject-predicate-object statements known as triples. For example, with a single RDF triple you can say that employee38 has a hireDate of "2020-04-03". A triple's predicate is a property specified with an IRI (an Internationalized version of a URI) to identify the namespace of the property name. A triple's subject and object can each be an IRI referencing any entity, and the object can also be a literal value such as the date 2020-04-03 or data of other types such as strings, numbers or Boolean values. </p>
```

Comment:
Oxford generally brings clarity...
```suggestion
<p>The RDF data model lets you state facts in three-part subject-predicate-object statements known as triples. For example, with a single RDF triple you can say that employee38 has a hireDate of "2020-04-03". A triple's predicate is a property specified with an IRI (an Internationalized version of a URI) to identify the namespace of the property name. A triple's subject and object can each be an IRI referencing any entity, and the object can also be a literal value such as the date 2020-04-03 or data of other types such as strings, numbers, or Boolean values. </p>
```